applications/StructuralMechanicsApplication/custom_processes/postprocess_eigenvalues_process.cpp

```diff
@@ -118,6 +118,12 @@ namespace Kratos
                     SizeType k = 0;
                     for (auto& r_dof : r_node_dofs)
                         r_dof.GetSolutionStepValue(0) = std::cos(angle) * r_node_eigenvectors(j,k++);
```

Comment:
also i would move the std::cos out of the loop

Reply:
ok makes sense, thx for the suggestions